streamlit_web.py

```diff
@@ -102,18 +104,23 @@ def retreive_data(data_retreive):
         input_assignments = st.sidebar.text_input(
                 "Enter path(s) to markdown documents (seperate by comma)"
         )
-    else:
+    elif data_retreive == "AWS":
         input_assignments = st.sidebar.text_input(
                 "Enter assignment names of the markdown \
 documents(seperate by comma)"
         )
         st.sidebar.info(
             "You will need to store keys and endpoints in the \
 environment variables")
+    else:
+        input_assignments = st.file_uploader("Choose a Markdown file",
+                                             type=['md'],
+                                             accept_multiple_files=True)
```

Comment:
An open question: is it possible/better to have the uploader appear in the sidebar instead of the main page?